tests/siteAliasTest.php

```diff
@@ -19,7 +19,7 @@ class saCase extends CommandUnishTestCase {
    */
   function testDispatchStrictOptions() {
     $aliasPath = UNISH_SANDBOX . '/site-alias-directory';
-    mkdir($aliasPath);
+    file_exists($aliasPath) ?: mkdir($aliasPath);
```

Comment:
Note, locally when running these, `UNISH_SANDBOX` was not torn down between each *function* in a class. I have no idea how these are passing on Travis or elsewhere since the sandbox is only cleaned up between each *class*.